packages/stress/src/demo.ts

```diff
@@ -35,45 +35,45 @@ async function spamInfo(count: number) {
         riverRegistry.getOperationalNodeUrls(),
     )
     for (let i = 0; i < count; i++) {
-        logger.info(`iteration ${i}`)
+        logger.debug(i, 'iteration')
```

Comment:
@miguel-nascimento if we care about seeing the "i" value in log metadata we need to make it a key-value pair e.g { iteration: i }

Reply:
ooh I forgot about this one